webdriver-spec.html

```diff
@@ -734,8 +734,9 @@
   to the description of <var>status</var> in the <a>status code registry</a>.
 
  <li><p>If <var>data</var> is not null, let <var>response</var>’s
-  <a>body</a> be the result of <a>serializing as JSON</a>
-  with <var>data</var> as the argument.
+  <a>body</a> be a JSON object with a top-level property
+  of <var>value</var>, the value of which is the result
```

Comment:
This should be `<code>value</code>` as it’s not a variable in the algorithm, but a key on the JSON Object.